tests/test_save.py

```diff
@@ -35,9 +39,9 @@ def run_simulation(save=False, dx=0.01, dy=0.01, a=0.5, timesteps=100):
     op.apply()
 
     if save:
-        return u.get_data(timesteps - 2)
+        return u.data[timesteps - 1, :]
     else:
-        return u.get_data()
+        return u.data[0, :]
```

Comment:
Do you want to use the `%` operator here to make sure this works even if the number of timesteps above is changed? That would also make the code more self-explanatory.